src/command.js

```diff
@@ -70,7 +70,7 @@ export default function () {
             userType = typeof userType === 'string'
                      ? userType
                      : (tags.subscriber ? 'subscriber' : 'viewer')
-            if (!(userLevels[userType] >= userLevels[opts.rank])) {
+            if (userLevels[userType] < userLevels[opts.rank]) {
```

Comment:
I'm pretty sure this was intentional. Might need an explanatory comment though lol. I think one of them could end up being `undefined`, allowing everyone to do everything, but it might not be relevant anymore